docker/xgb/centos/common/sbin/install_packages

```diff
@@ -13,4 +13,5 @@ rpm --rebuilddb
 yum -y install https://repo.ius.io/ius-release-el$(rpm -q --queryformat '%{VERSION}' centos-release).rpm
 sed -i '/\[ius.*\]/a sslverify=0' /etc/yum.repos.d/ius.repo
 rpm --rebuilddb
-yum -y install python35u-devel python-pip python-virtualenv git mysql-devel unzip make tar
+yum -y install python35u-devel python-pip python-virtualenv git mysql-devel unzip make tar zlib-devel libffi-devel
```

Comment:
Good one, seem to me as legacy code. The 3.7 is installed outside of this script e.g. [here](https://github.com/h2oai/h2o-3/pull/6664/files#diff-15e94164bde3dd6d7ec3c458f612d50e444a897ea31a23a5ac3488a4f0c4412dR28) I will put FIXME comment for future Adam to remove it, since it is already inside of the docker image and if I fix it here its still has no effect on anything.